### PR TITLE
created servlet charged with adding images

### PR DESCRIPTION
**Pull request by InfinityGrip**

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/ImagesServlet.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        String projectId = "spring21-sps-2";
+        String bucketName = "spring21-sps-2";
+        Storage storage = StorageOptions.newBuilder().setProjectId(projectId).build().getService();
+        Bucket bucket = storage.get(bucketName);
+        Page<Blob> blobs = bucket.list();
+        
+
+
+        response.setContentType("text/html;");
+        for (Blob blob : blobs.iterateAll()){
```
Comment (Corramel) [inline src/main/java/com/google/sps/servlets/ImagesServlet.java:29]: This works great, but just for reference we can also do it in JSON which might be easier. I've done the implementation on my portfolio, and it's not too bad. I could handle it if it need be. Though not a real issue for now.

**Review by Corramel — APPROVED** [review]
Looks good! Good job.

File: src/main/java/com/google/sps/servlets/ImagesServlet.java (hunk 1 above)
Comment (matty377) [reply src/main/java/com/google/sps/servlets/ImagesServlet.java:29]: Good call. Feel free to create a task for that in Trello.

File: src/main/java/com/google/sps/servlets/ImageFormServlet.java [hunk 2]
```diff
@@ -16,24 +22,46 @@
...
-    
     @Override
     /**
      * @param request Received request
      * @param response Outgoing response
      */
     public void doPost(HttpServletRequest request, HttpServletResponse response) 
         throws ServletException, IOException {
-            UUID uuid = UUID.randomUUID(); //This is a randomly generated ID for the image
+            //UUID uuid = UUID.randomUUID(); //This is a randomly generated ID for the image
```
Comment (matty377) [inline src/main/java/com/google/sps/servlets/ImageFormServlet.java:32]: Can delete this commented out line.

File: src/main/java/com/google/sps/servlets/ImageFormServlet.java [hunk 3]
```diff
@@ -16,24 +22,46 @@
...
 
             String message = request.getParameter("message");
             Part filePart = request.getPart("image");
-            String fileName = uuid.toString(); //This could be changed later if we have a system for it
+            //String fileName = uuid.toString(); //This could be changed later if we have a system for it
```
Comment (matty377) [inline src/main/java/com/google/sps/servlets/ImageFormServlet.java:36]: And this one.

**Review by matty377 — APPROVED** [review]

**Comment by matty377** [comment]
LGTM.

**Pull request merged**